mmdt/mmdt/pyvenv.cfg

```diff
@@ -0,0 +1,5 @@
+home = C:\Program Files\Python312
+include-system-site-packages = false
+version = 3.12.1
+executable = C:\Users\Acer\Desktop\project\mmdt_web_app\mmdt\Scripts\python.exe
+command = C:\Users\Acer\Desktop\project\mmdt_web_app\mmdt\Scripts\python.exe -m venv C:\Users\Acer\Desktop\project\mmdt_web_app\mmdt\mmdt
```

Comment:
I think we don't need this one in code base, that is local setting , please add in .`gitignore` too.

Reply:
- [ ] I was fixed this comment.